recipe-client-addon/lib/PreferenceExperiments.jsm

```diff
@@ -75,6 +92,20 @@ CleanupManager.addCleanupHandler(() => PreferenceExperiments.stopAllObservers())
 
 this.PreferenceExperiments = {
   /**
+   * Set the default preference value for active experiments that use the
+   * default preference branch.
+   */
+  async init() {
+    const store = await ensureStorage();
+    for (const experiment of Object.values(store.data)) {
```

Comment:
It ends up only saving a few characters. I think we use `for/of` in most other places over `forEach` as well, but I might be misremembering.